JUSTFILE

```diff
@@ -6,24 +6,52 @@ install-kind:
 
 setup-env: install-kind create-kind-cluster
 
+# Vanilla
+
+build-deploy-cloudbeat: build-cloudbeat load-cloudbeat-image delete-cloudbeat deploy-cloudbeat
+
+build-deploy-cloudbeat-debug: build-cloudbeat-debug load-cloudbeat-image delete-cloudbeat-debug deploy-cloudbeat-debug
+
 load-cloudbeat-image:
   kind load docker-image cloudbeat:latest --name kind-mono
 
 build-cloudbeat:
   GOOS=linux go build -v && docker build -t cloudbeat .
 
 deploy-cloudbeat:
-  kubectl delete -f deploy/k8s/cloudbeat-ds.yaml -n kube-system & kubectl apply -f deploy/k8s/cloudbeat-ds.yaml -n kube-system
-
-build-deploy-cloudbeat: build-cloudbeat load-cloudbeat-image deploy-cloudbeat
+  kubectl apply -f deploy/k8s/cloudbeat-ds.yaml -n kube-system
 
 build-cloudbeat-debug:
   GOOS=linux CGO_ENABLED=0 go build -gcflags "all=-N -l" && docker build -f Dockerfile.debug -t cloudbeat .
 
 deploy-cloudbeat-debug:
-  kubectl delete -f deploy/k8s/cloudbeat-ds-debug.yaml -n kube-system & kubectl apply -f deploy/k8s/cloudbeat-ds-debug.yaml -n kube-system
+   kubectl apply -f deploy/k8s/cloudbeat-ds-debug.yaml -n kube-system
```

Comment:
a few points here:
- as mentioned in the previous comment, I think there shouldn't be a separation between vanilla & eks yaml files so the same thing goes for the just commands.
- I think login-aws & push-ecr can be unified to one command
- U shouldn't use the latest tag to push to ecr, since you'll override what master pushed(I suggest using branch/pr ref for the tag instead).
- I'd unify them and add a [conditional](https://github.com/casey/just#conditional-expressions) to the just recipe . for exp `if env_var("EKS") == "true"` then invoke the EKS relevant step(publish-ecr) instead of kind-load-image.